resources/scripts/scrape/ipv4info.ads

```diff
@@ -86,8 +66,8 @@ function next_page(ctx, domain, resp, page)
         return ""
     end
 
-    local u = "http://ipv4info.com" .. match[1]
-    local page, err = request(ctx, {['url']=u})
+    local url = "http://ipv4info.com" .. match[1]
+    local page, err = request(ctx, {url=url})
```

Comment:
We cannot have `url=url` and should be keeping the square brackets `['url']`

Reply:
Done